data/json/items/armor/torso_armor.json

```
@@ -268,6 +316,35 @@
     "material_thickness": 4,
     "material": [ "leather", "bone" ]
   },
+  {
+    "id": "jacket_leather_tire",
+    "repairs_like": "jacket_leather",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "tire leather armour" },
```

```suggestion
    "name": { "str_sp": "tire leather armor" },
```
American spelling, and I think the plural is also armor?